platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformExecutionBuildOperationIntegrationTest.groovy

```diff
@@ -216,6 +216,10 @@ class ArtifactTransformExecutionBuildOperationIntegrationTest extends AbstractIn
                 @InputArtifact
                 abstract Provider<FileSystemLocation> getInputArtifact()
 
+                // We need an incremental transform to have a project-bound workspace
+                @Inject
+                abstract InputChanges getInputChanges()
```

Comment:
❓ Again, are we losing some coverage here?

Reply:
Added a `where:` clause.